packages/web/src/components/Plan.tsx

```diff
@@ -105,23 +79,57 @@ export class Plan extends React.Component<{}, State> {
 
     constructor(props: {}) {
         super(props);
+
+        const theData = new EventData(
+            1,
+            "Tutorial Event",
+            new Date(),
+            new Date(),
+            "This is an example event! You can add or change descriptions and titles by clicking the edit button at the side. + search help when it's implemented, etc. etc."
+        );
+        this.state.events.push(theData);
+
+        console.log("The EventData is " + theData);
+        console.log("The EventData id is " + theData.id);
+        console.log("The EventData name is " + theData.name);
+        console.log("The EventData description is " + theData.description);
+        console.log("The Elements in the Array is " + this.state.events);
+        console.log("The No. of Elements is " + this.state.events.length);
     }
 
     componentDidMount(): void {
         const restoredData = restoreData();
         const restoredPlan = restorePlan();
+        const newData = new EventData(
+            1,
+            "Tutorial Event",
+            new Date(),
+            new Date(),
+            "This is an example event! You can add or change descriptions and titles by clicking the edit button at the side. + search help when it's implemented, etc. etc."
+        );
+
+        const arr = [newData];
 
         if (restoredPlan) {
             console.log("Restored plan from browser");
         }
 
-        // if no restored state, use an empty plan and default title
+        // if no restored state, revert to launch tutorial
         this.setState({
             data: restoredData
                 ? { title: restoredData.title }
                 : { title: "Your Plan" },
-            events: restoredPlan ? restoredPlan : [],
+            events: restoredPlan ? restoredPlan : arr,
```

Comment:
This should be the correct way to add the tutorial event :+1:

Reply:
I tried that yesterday but I don't think the tutorial event appears on the localhost page; it seemed like the object wasn't getting pushed onto the array

Reply:
I think in the newest commit I made a addTutorial function in the PlanHelpers.ts (modified addEvent) but only calls in the componentDidMount()